sdk/servicebus/service-bus/src/sender.ts

```diff
@@ -247,13 +247,12 @@ export class SenderImpl implements Sender {
       return this._sender.sendBatch(batch, options);
     } else if (isServiceBusMessageBatch(messageOrMessagesOrBatch)) {
       return this._sender.sendBatch(messageOrMessagesOrBatch, options);
+    } else if (messageOrMessagesOrBatch != null && typeof messageOrMessagesOrBatch === "object") {
```

Comment:
What about an object that is not of type ServiceBusMessage?
isServiceBusMessage() - check for the body param.
I think we should check if the body is defined or not at this point.

Reply:
Isn't `body` optional? A user could just set the label and user properties right? (I'm not sure how useful that is, but it's something I can do in track1)

Reply:
Currently, body attribute is not optional, typescript would complain.
Though it is not optional, we don't verify if it was provided or not in the SDK.

Reply:
Yeah, it's simple enough to at least say they've set the 'body' attribute even if they're not actually required to set anything aside from `undefined`.

Reply:
Done!